pocket/app.js

```diff
@@ -204,6 +204,16 @@ app.post('/api/stake', (req, res) => {
     res.send(response);
 })
 
+app.post('/api/unstake', (req, res) => {
+    // res.send({});
+    // return;
+    const network = shell.exec(`echo $NETWORK`).stdout.trim();
+    const passphrase = shell.exec(`echo $KEYFILE_PASSPHRASE`).stdout.trim();
+    const address = shell.exec(`pocket accounts list --datadir=/home/app/.pocket/ | cut -d' ' -f2- `).stdout.trim();
+    const response = shell.exec(`pocket nodes unstake ${address} ${address} ${network} 10000 false --datadir=/home/app/.pocket/ --pwd "${passphrase}" | tail -n +3`).stdout.trim();
```

Comment:
```suggestion
    const response = shell.exec(`pocket nodes unstake ${address} ${address} ${network} 10000 false --datadir=/home/app/.pocket/ --pwd '${passphrase}' | tail -n +3`).stdout.trim();
```

Reply:
this is the problem we had with passwords with characters like `!` in a passphrase when passed as a variable its treated as an event escape error and prints the remainder of the password to the console.   where it would do history or variable expansion.  Single quotes will use all contents as a literal string perfect for passwords, and it works